trin-core/src/utp/stream.rs

```diff
@@ -58,10 +58,18 @@ const BASE_HISTORY: usize = 10; // base delays history size
 const MAX_BASE_DELAY_AGE: Delay = Delay(60_000_000);
 // Discv5 socket timeout in milliseconds
 const DISCV5_SOCKET_TIMEOUT: u64 = 25;
+/// Process uTP streams interval in milliseconds
+// const PROCESS_UTP_STREAMS_INTERVAL: u64 = 20;
 
 /// uTP connection id
 type ConnId = u16;
 
+/// uTP payload data
+pub type UtpPayload = Vec<u8>;
+
+/// UtpListener unbounded receiver for emitted events
+pub type UtpListenerUnboundedReceiver = Arc<RwLock<UnboundedReceiver<UtpListenerEvent>>>;
```

Comment:
I sort of prefer no alias, so that it's clear that we are passing around a `Arc<RwLock<..>>`, and because we only use this once as far as I can tell. Just a personal preference though.

Reply:
Hmm, that's annoying. maybe we can use

```rust
pub type UnboundedUtpEventReceiver = UnboundedReceiver<UtpListenerEvent>;
```

and then use `Arc<RwLock<UnboundedUtpEventReceiver>>`.

If not, then this is okay.

Reply:
Actually, we don't need to wrap this in Arc<RwLock<..>> anymore, so I removed the alias :partying_face:.